libs/integration-tests-ocm/src/integration/ui-behaviour/cluster-creation.spec.ts

```diff
@@ -22,26 +22,28 @@ describe('Assisted Installer UI behaviour - cluster creation', () => {
     it('Should have the correct values for the Openshift versions', () => {
       commonActions.visitNewClusterPage();
 
-      clusterDetailsPage.getSelectedOpenShiftVersion() .should(
-        'contain',
-        `OpenShift ${versionsFixtures.getDefaultOpenShiftVersion()}`,
-      );
+      clusterDetailsPage
+        .getSelectedOpenShiftVersion()
+        .should('contain', `OpenShift ${versionsFixtures.getDefaultOpenShiftVersion()}`);
 
       // Checking that the submitting value (item ID) for each version is correct
       clusterDetailsPage.openOpenshiftVersionDropdown();
 
       const expectedVersionIds = versionsFixtures.getExpectedVersionIds();
-      clusterDetailsPage.getOpenshiftVersionDropdown().find('[role="menuitem"]').each((versionItem, index) => {
-        expect(versionItem).to.have.id(expectedVersionIds[index]);
-      });
+      clusterDetailsPage
+        .getOpenshiftVersionDropdown()
+        .find('[role="menuitem"]')
+        .each((versionItem, index) => {
```

Comment:
As per the comment in line 29, the purpose of this test is to make sure the submit value for each version is the expected one, as we've had problems in the past with that (since sometimes the values are in the format 4.10, and others in the format 4.10.3).

So I'd maintain the check for the "ID" (which corresponds to the submit value), and additionally, you can add the check for the name.